### PR TITLE
Csg/feature/qr code validation

### PR DESCRIPTION
**Pull request by Xiole-KKE**
#### Problem
Expose QR code parsing from RPC server.

 #### Summary of Changes
One function that return the parsed QR code information with a dictionary object.
Exposed API added for RPC server

 Fixes chip-csg/chip-certification-tool#160

### CODE REVIEW

File: src/controller/python/chip/setup_payload/setup_payload.py
Comment (wahajsyed) [inline src/controller/python/chip/setup_payload/setup_payload.py:98]: Should we add a demarcator  here to 'contain' our changes so we can work only in this area in the future, like this or 🤷 : 
######--------------------------------------------------######

File: src/controller/python/chip/setup_payload/setup_payload.py
Comment (wahajsyed) [inline src/controller/python/chip/setup_payload/setup_payload.py:104]: No new line at the end of the file

File: src/controller/python/chip/setup_payload/setup_payload.py
Comment (Xiole-KKE) [reply src/controller/python/chip/setup_payload/setup_payload.py:98]: Sure! Keep our change ours.

File: src/controller/python/chip/setup_payload/setup_payload.py
Comment (Xiole-KKE) [reply src/controller/python/chip/setup_payload/setup_payload.py:104]: Will add one, coding without lint is tricky.

**Comment by wahajsyed** [comment]
@nirav1911  has added a neat way of sending pack data with an error code [here](https://github.com/chip-csg/connectedhomeip/pull/19/files#)

We could
1. Merge your PR with a Todo  and file and issue to track updating the RPC response
2. Wait for his changes to merge so you can follow the same pattern

**Review by wahajsyed — APPROVED** [review]
LGTM with minor comments

**Comment by Xiole-KKE** [comment]
We can merge this PR later. Let's wait for Nirav's change

**Review by cjandhyala — APPROVED** [review]

File: src/controller/python/chip-device-ctrl.py
Comment (Xiole-KKE) [inline src/controller/python/chip-device-ctrl.py:639]: This is part of Nirav's change, it will disappear once Nirav's change is merged into test_event2.2 and I do a merge onto that.

**Comment by wahajsyed** [comment]
@Xiole-KKE  Can we get this rebased and merged?

**Pull request merged**
